src/components/Card/Card.scss

```diff
@@ -22,6 +22,11 @@
   @include page-content-when-not-fully-condensed {
     padding: spacing(loose) spacing(loose) 0;
   }
+
+  + .Section-subdued {
+    border-top: border-width() solid color('sky');
+    margin-top: spacing(loose);
+  }
```

Comment:
This is styling a subdued section, should these rules be nested under the `.Section` selector instead?

```scss
.Section {
   ...

   .Header + & {
     ...
  }
}

Reply:
That was fast! Thank you.

The `>` is the key here. What it's doing is targeting the subdued section only if it's the next sibling of subdued. Moving it out wouldn't achieve that.

Reply:
I'm sorry, Ryan. I tried your suggestion and you're absolutely right. 👍

Reply:
I do have mixed feelings about doing it this way. Nesting with `+ &` is hard to understand. By contrast, the CSS that it compiles to is plain as day: `.Header + .Section {…}`.

Separately, it’s really helpful to have all the styles for each element (in this case, all the styles for `.Section`) in one place in the file.

My personal preference would be to have both, by avoiding the `+ &` contortions but keeping the styles together:

```scss
.Section {
  // All the basic styles for the section
}

.Header + .Section {
  // Specific styles for this case right below
}
``` 

However, I think our in-house Sass style prefers nesting, so we need to do the `+ &` thing unless we want to open a discussion about code style.

Reply:
Yes, I think the linter complains if you repeat a class.